### ISSUE TITLE
FEL not working - Error included

### ISSUE THREAD

**Issue by SculpturatusLabs** [veg/hyphy#1703]
FEL algorithm was working this morning and now does not work even with the same exact .nex file that I processed this morning. I get the following error message:

/var/spool/torque/mom_priv/jobs/3854981.new-silverback.SC: line 37: mpirun: command not found

Using Mac Sonoma 14.4.1 - updated operating system in between the last time it worked and first failure. Using Brave Browser (worked flawlessly in the past). I have a feeling this is an issue with the new MacOS?

**Comment by stevenweaver** [comment]
Dear @SculpturatusLabs, 

Thanks, in an attempt to resolve occasional issues where Datamonkey is reporting module conflicts (which has no bearing on outcome, but confuses users when they see it), I commented out a line where we are explicitly loading modules in the script. However, there is heterogeneity among nodes when it comes to which modules are being loaded. I've reverted my change, and will need to look at why module conflicts are happening more closely at a later time. 

tl;dr -- the site should be functional again.

Best,
Steven
